tests/test_config.py

```diff
@@ -168,3 +167,20 @@ def test_not_found_default_connection_from_evn_variable(test_root_path):
         get_default_connection()
 
     assert ex.value.message == "Connection not_existed_connection is not configured"
+
+
+def test_connections_toml_override_config_toml(test_snowcli_config, snowflake_home):
+    from snowflake.cli.api.config import CONFIG_MANAGER
+
+    connections_toml = snowflake_home / "connections.toml"
+    connections_toml.write_text(
+        """[default]
```

Comment:
```suggestion
        """[something_other_than_default]
```
My personal preference is to avoid something that can be default 🙈